Rakefile

```diff
@@ -69,6 +91,31 @@ task :release => [:install, :shrinkwrap, :pack] do
   puts "Create a new #{version} release on github.com and upload the #{name} tarball"
   puts 'You can find directions here: https://github.com/blog/1547-release-your-software'
   puts 'Make sure you add release notes!'
+
+  cp ::File.join(base_dir, "#{name}-#{version}.tgz"), pkg_dir
+
+  begin
+    latest_release = github_client.latest_release(github_repo)
+  rescue Octokit::NotFound
+    latest_release = OpenStruct.new(name: 'master')
+  end
+
+  release = github_client.create_release(
+    github_repo,
+    "v#{version}",
+    :name => "v#{version}",
+    :draft => true
+  )
```

Comment:
Will this fail if the release already exists?  Such as, in the event the version was not bumped.